model/src/main/java/com/basistech/rosette/apimodel/batch/BatchRequestItem.java

```diff
@@ -0,0 +1,67 @@
+/*
+* Copyright 2016 Basis Technology Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package com.basistech.rosette.apimodel.batch;
+
+import com.basistech.rosette.apimodel.NameSimilarityRequest;
+import com.basistech.rosette.apimodel.NameTranslationRequest;
+import com.basistech.rosette.apimodel.Request;
+
+/**
+ * One item in a batch request. Each item consists of an input
+ * to process and an endpoint to apply to the item.
+ * The inputs are specified by {@link com.basistech.rosette.apimodel.Request} objects.
+ * The special content type {@code application/vnd.basistech-multiple-inputs}
+ * used in {@link Request#contentType} identifies
+ * a json file consisting of an array of strings; each string being treated as an
+ * input. The string length in this case is limited to 200 characters.
+ */
+public class BatchRequestItem {
```

Comment:
We don't have an ID field in the Request class, we either need to add to that or here.